scripts/deploy/package-webapi.ps1

```diff
@@ -22,13 +22,19 @@ param(
 
     [string]
     # Version to give to assemblies and files.
-    $Version = "1.0.0",
+    $Version = "0.0.0",
 
     [string]
     # Additional information given in version info.
-    $InformationalVersion = ""
+    $InformationalVersion = "",
+    
+    [bool]
+    # Whether to build frontend files (true by default)
+    $BuildFrontendFiles = $true
```

Comment:
it seems like we should align this flag with the one in `.sh`?

Reply:
Done